app/models/gsa18f/procurement.rb

```diff
@@ -10,6 +10,13 @@ class Procurement < ActiveRecord::Base
     URGENCY = DATA['URGENCY']
     OFFICES = DATA['OFFICES']
     RECURRENCE = DATA['RECURRENCE']
+    PURCHASE_TYPES = {
+      "Software" => 0,
+      "Training/Event" => 1,
+      "Office Supply/Miscellaneous" => 2
+    }
```

Comment:
Since this is for activerecord enums and is using standard enum functionality, I would assume that people adding to the codebase would know that these integers are in the db / should not be changed if they know how AR enums work. Do you think that is not the case?